orchagent/swssnet.h

```diff
@@ -76,6 +76,28 @@ inline static sai_ip_prefix_t& copy(sai_ip_prefix_t& dst, const IpAddress& src)
     return dst;
 }
 
+inline static sai_ip_prefix_t& copy(sai_ip_prefix_t& dst, const IpAddress& src, const IpAddress& mask)
+{
+    auto sip = src.getIp();
+    auto msk = mask.getIp();
+    switch(sip.family)
+    {
+        case AF_INET:
+            dst.addr_family = SAI_IP_ADDR_FAMILY_IPV4;
+            dst.addr.ip4 = sip.ip_addr.ipv4_addr;
+            dst.mask.ip4 = msk.ip_addr.ipv4_addr;
+            break;
+        case AF_INET6:
+            dst.addr_family = SAI_IP_ADDR_FAMILY_IPV6;
+            memcpy(dst.addr.ip6, sip.ip_addr.ipv6_addr, 16);
```

Comment:
Use a define instead, something like `IPV6_ADDRESS_LEN` and use it for the other function as well. People might double guess what 16 stands for.